app/code/Magento/Tax/Block/Sales/Order/Tax.php

```diff
@@ -260,13 +270,19 @@ protected function _initShipping()
     }
 
     /**
+     * Init discount.
+     *
+     * phpcs:disable Magento2.CodeAnalysis.EmptyBlock
+     *
      * @return void
      */
     protected function _initDiscount()
     {
     }
 
```

Comment:
Let's reenable empty block sniff after the method
```suggestion
    //phpcs:enable
```